source/manual/howto-change-a-government-following-an-election.html.md

```diff
@@ -46,39 +57,29 @@ The new government should also instantly appear in the response from
 
 [historic]: https://github.com/alphagov/whitehall/blob/e518218355d158bfff036a02e312dda714da0aa6/app/models/edition.rb#L647
 
 Marking content as political can happen automatically through the
-[PoliticalContentIdentifier][]. The `political` flag on an `Edition`
-can also be set manually.
+[PoliticalContentIdentifier][]. The `political` flag on an `Edition` can also
+be set manually.
 
 [PoliticalContentIdentifier]: https://github.com/alphagov/whitehall/blob/master/lib/political_content_identifier.rb
 
-#### Updating newly historic content
-
-There is a Rake task to republish all political content and ensure
-these banners are applied for political content which is now
-associated with a previous government..
-
-[election:republish_political_content](https://deploy.integration.publishing.service.gov.uk/job/run-rake-task/parambuild/?delay=0sec&TARGET_APPLICATION=whitehall&MACHINE_CLASS=whitehall_backend&RAKE_TASK=election:republish_political_content)
-
-In Integration, this will republish around 92,000 documents. The
-Publishing API may take between 4 and 6 hours to process these.
-
-Grafana monitoring for:
-
-* [Publishing API](https://grafana.integration.publishing.service.gov.uk/dashboard/file/publishing-api.json?refresh=5s&orgId=1&from=now-6h&to=now)
-* [Sidekiq](https://grafana.integration.publishing.service.gov.uk/dashboard/file/sidekiq.json?refresh=1m&orgId=1&from=now-6h&to=now&var-Application=whitehall&var-Queues=All)
+#### Reindexing political content in search
 
-This may trigger Icinga alerts for the Publishing API:
+There is a Rake task to reindex all political content in search. This must be
+done after a government is closed.
 
-* [Established connections for publishing-api exceeds 8](https://graphite.integration.govuk.digital/render/?width=1000&height=600&colorList=red,orange,blue,green,purple,brown&target=alias%28dashed%28constantLine%2810%29%29,%22critical%22%29&target=alias%28dashed%28constantLine%288%29%29,%22warning%22%29&target=publishing_api-ip-10-1-4-39_eu-west-1_compute_internal.tcpconns-3093-local.tcp_connections-ESTABLISHED)
-* [high load on](https://grafana.integration.govuk.digital/dashboard/file/machine.json?refresh=1m&orgId=1&var-hostname=publishing_api-ip-10-1-4-39_eu-west-1_compute_internal)
+[Run `search:political` in Integration](https://deploy.integration.publishing.service.gov.uk/job/run-rake-task/parambuild/?TARGET_APPLICATION=whitehall&MACHINE_CLASS=whitehall_backend&RAKE_TASK=search:political)
+[Run `search:political` in Staging](https://deploy.blue.staging.govuk.digital/job/run-rake-task/parambuild/?TARGET_APPLICATION=whitehall&MACHINE_CLASS=whitehall_backend&RAKE_TASK=search:political)
+[⚠️ Run `search:political` in Production ⚠️](https://deploy.publishing.service.gov.uk/job/run-rake-task/parambuild/?TARGET_APPLICATION=whitehall&MACHINE_CLASS=whitehall_backend&RAKE_TASK=search:political)
```

Comment:
Could we add line breaks to these please?

Reply:
Yep! I've amended the commit.